website/hooks/use-cta.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from "react"
+import useragent from "express-useragent"
+import { getCtaDestination } from "utils/get-cta-destination"
+
+export const useCta = (): string => {
```

Comment:
nicely done. Typing return should be unnecessary here as typescript will correctly deduce type from return value.